utils/mvn/utils.go

```diff
@@ -24,11 +25,12 @@ const (
 	mavenExtractorDependencyVersion = "2.30.2"
 	classworldsConfFileName         = "classworlds.conf"
 	MavenHome                       = "M2_HOME"
```

Comment:
MavenHome --> mavenHome

Reply:
+ I'd prefer to name it m2_home, because MAVEN_HOME is a different variable